web/html/src/utils/data-providers/page-control.ts

```diff
@@ -6,14 +6,14 @@ type SortType = {
 export default class PageControl {
   page: number;
   pageSize: number;
-  query: string | null | undefined;
-  sort: SortType | null | undefined;
+  query?: string | null;
+  sort?: SortType | null;
```

Comment:
so is this equivalent in functionality?

Reply:
@cbbayburt It's not quite equivalent, but in the above case that's how it was used. As for the difference, `arg: T | undefined` means the arg has to be specified, even if `undefined`:

```ts
function foo(arg: string | undefined) {...}
foo("bar"); // valid
foo(undefined); // valid
foo(); // NOT valid
```

Making the arg optional means that both `undefined` and omitting the arg is fine:  

```ts
function foo(arg?: string) {...}
foo("bar"); // valid
foo(undefined); // valid
foo(); // valid
```

Reply:
ok so TS will know the difference. Thanks :+1: